src/main/java/com/g4s8/ghman/bot/BotApp.java

```diff
@@ -55,13 +34,11 @@
  * Telegram bot entry point.
  *
  * @since 1.0
- * @todo #1:30min Refactor this class:
- *  move actual logic to separate takes and keep only
- *  composition structure in this class.
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  * @checkstyle LineLengthCheck (500 lines)
  * @checkstyle ExecutableStatementCountCheck (500 lines)
  */
+@SuppressWarnings("PMD.UnnecessaryFullyQualifiedName")
```

Comment:
@samihibrahim why is this needed?

Reply:
@victornoel this was already present in the original class. I added a todo to this classes to fix both of these exclusions.